server/sonar-web/src/main/js/apps/project-admin/store/actions.js

```diff
@@ -47,9 +47,14 @@ export const receiveProjectProfiles = (projectKey, profiles) => ({
   profiles
 });
 
-export const fetchProjectProfiles = projectKey =>
+export const fetchProjectProfiles = (projectKey, organization) =>
   dispatch => {
-    Promise.all([getQualityProfiles(), getQualityProfiles({ projectKey })]).then(responses => {
+    Promise.all([
+      organization ? getQualityProfiles({ organization }) : getQualityProfiles(),
```

Comment:
Ok I'll do it, do you know if there could be particular cases where we intentionally want some `null` or `undefined` values in our api calls ?